client/js/table-model.js

```diff
@@ -16,6 +16,21 @@ class TableModel {
   setValue(location, value) {
     this.data[this._getCellId(location)] = value;
   }
+
+  getSum(column) {
+    let sum = [];
+    for (var prop in this.data) {
```

Comment:
Please do not manipulate `this.data` directly.  What if we decide later to change the data type of `data` from an object to an array-of-arrays?  Then all this code breaks.  Whereas if you used the interface that your model provides, your code will not have to change.  This is called *separation of concerns* AKA *decoupling*.  Don't couple to the internals if you can help it.

The interface your model provides for this is called `getValue`.